src/script/auth/module/action/InvitationAction.ts

```diff
@@ -52,22 +52,19 @@ export class InvitationAction {
       };
 
       const teamId = selfSelector.getSelfTeamId(state);
-      return Promise.resolve()
-        .then(() => apiClient.teams.invitation.api.postInvitation(teamId, newInvite))
-        .then(createdInvite => {
-          dispatch(InvitationActionCreator.successfulAddInvite(createdInvite));
-        })
-        .catch(error => {
-          dispatch(InvitationActionCreator.failedAddInvite(error));
-          throw error;
-        });
+      try {
+        const createdInvite = await apiClient.teams.invitation.api.postInvitation(teamId, newInvite);
+        dispatch(InvitationActionCreator.successfulAddInvite(createdInvite));
+      } catch (error_1) {
```

Comment:
Hold your horses :horse: 

```suggestion
      } catch (inviteError) {
```